lib/jormungandr/src/Cardano/Wallet/Jormungandr/Compatibility.hs

```diff
@@ -69,8 +75,14 @@ genesis = BlockHeader
 instance TxId (Jormungandr n) where
     txId = undefined
 
-instance KeyToAddress (Jormungandr n) where
-    keyToAddress = undefined
+instance KnownNetwork n => KeyToAddress (Jormungandr n) where
+    keyToAddress =
+        Address . toStrict . runPut . putAddress . xpubPublicKey . getKey
+      where
+          putAddress addr = do
+              -- NOTE: only single address supported for now
+              putWord8 (single @n)
+              putByteString addr
```

Comment:
As discussed on @Anviking's PR, it would be best to keep the binary stuff inside the Binary module and move `putAddress` there :+1:

Reply:
It can be here for the time being, but I think my PR should be merged soon fixing this.

Reply:
it is temporary as mentioned in the bullet points of description, I will wait for @Anviking PR to be merged into master and will take the fix as it is